src/Microsoft.PowerShell.Commands.Utility/commands/utility/ShowMarkdownCommand.cs

```diff
@@ -28,143 +30,192 @@ public class ShowMarkdownCommand : PSCmdlet
-                            ErrorCategory.InvalidData,
-                            html);
+                            e,
+                            "ErrorWritingTempFile",
+                            ErrorCategory.WriteError,
+                            tmpFilePath);
 
                         WriteError(errorRecord);
+                        return;
                     }
-                }
-                else
-                {
-                    var vt100String = markdownInfo.VT100EncodedString;
 
-                    if (!string.IsNullOrEmpty(vt100String))
+                    if (InternalTestHooks.ShowMarkdownOutputBypass)
                     {
-                        if (InternalTestHooks.ShowMarkdownOutputBypass)
-                        {
-                            WriteObject(vt100String);
-                            return;
-                        }
+                        WriteObject(html);
+                        return;
+                    }
 
-                        if (stepPipe != null)
-                        {
-                            stepPipe.Process(vt100String);
-                        }
+                    try
+                    {
+                        ProcessStartInfo startInfo = new ProcessStartInfo();
+                        startInfo.FileName = tmpFilePath;
+                        startInfo.UseShellExecute = true;
+                        Process.Start(startInfo);
                     }
-                    else
+                    catch (Exception e)
                     {
-                        string errorMessage = StringUtil.Format(ConvertMarkdownStrings.MarkdownInfoInvalid, "VT100EncodedString");
                         var errorRecord = new ErrorRecord(
-                            new InvalidDataException(errorMessage),
-                            "VT100EncodedStringIsNullOrEmpty",
-                            ErrorCategory.InvalidData,
-                            vt100String);
+                            e,
+                            "ErrorLaunchingDefaultApplication",
+                            ErrorCategory.InvalidOperation,
+                            targetObject: null);
 
                         WriteError(errorRecord);
+                        return;
                     }
                 }
+                else
+                {
+                    string errorMessage = StringUtil.Format(ConvertMarkdownStrings.MarkdownInfoInvalid, "Html");
+                    var errorRecord = new ErrorRecord(
+                        new InvalidDataException(errorMessage),
+                        "HtmlIsNullOrEmpty",
+                        ErrorCategory.InvalidData,
+                        html);
+
+                    WriteError(errorRecord);
+                }
             }
             else
             {
-                string errorMessage = StringUtil.Format(ConvertMarkdownStrings.InvalidInputObjectType, inpObj.GetType());
```

Comment:
> ConvertMarkdownStrings.InvalidInputObjectType

I guess this resource string can be removed now.

Reply:
That string is still used by `ConvertFrom-Markdown` and `Set-MarkdownOption` cmdlets